src/support/data/shoppingLists.ts

```diff
@@ -68,6 +68,16 @@ export const allShoppingLists: ShoppingList[] = [
     created_at: new Date('2023-02-12T15:17:33'),
     updated_at: new Date('2023-02-12T15:17:33'),
   },
+  {
+    id: 6,
+    game_id: 77,
+    aggregate_list_id: 3,
+    aggregate: false,
+    title: 'Hjerim',
+    list_items: emptyListItems,
```

Comment:
We want to have one list without list items for this game for testing/Storybook purposes.